app/views/products/show.html.erb

```diff
@@ -37,6 +37,8 @@
   <% if @product.highest_rating_comment.nil? %>
     Sorry, there are no comments about the product. <br>Be the first writing a review!
   <% else %>
+    <div class="product-reviews">
+      <div class="alert alert-info" style="display:none" role="alert">A new comment was posted!</div>
```

Comment:
why not to have a class called hidden, and that class has the display none property, instead of having inline styling (which is less manageable)?